docs/api/sql/Function.md

```diff
@@ -1216,6 +1216,38 @@ Output:
 POLYGON ((0 0, 0 3, 1 3, 1 0, 0 0))
 ```
 
+## ST_Expand
+
+Introduction: Returns a geometry expanded from the bounding box of the input. The expansion can be specified in two ways:
```

Comment:
Please say this only deal with X, Y, Z value.

If `uniformDelta` is used and the geometry has `Z`, all X, Y, Z will be expanded. If only `deltaX` and `deltaY` are specified, only X and Y will be expanded even if the geometry has `Z`